src/components/Homepage/HomePage.js

```diff
@@ -0,0 +1,13 @@
+import Banner from './Banner'
+import Topics from './Topics'
+
+
+function HomePage(){
+  return(
+    <>
```

Comment:
This should a div instead of a fragment as to allow for top level css for the page. 
It would be best to use fragments when the component is to be embedded.

Reply:
Okay, I will change react fragment into div.